src/components/BaseComponents/TextInput/TextInput.tsx

```diff
@@ -46,6 +46,10 @@ export default function TextInput(props) {
     inputProps['aria-describedby'] = describedByIDs;
   }
 
+  if (!inputProps['autoComplete']) {
+    inputProps['autoComplete'] = 'off';
+  }
+
```

Comment:
if (configAlternateDesignSystem?.autocomplete) {
    extraInputProps['autoComplete'] = configAlternateDesignSystem.autocomplete;
  }

This logic is already written in override TextInput so you can add else condition here.